x/docs/internal/keeper/keeper_test.go

```diff
@@ -440,16 +488,16 @@ func TestKeeper_GetUsersSet_EmptySet(t *testing.T) {
 func TestKeeper_SetUserDocuments(t *testing.T) {
 	cdc, ctx, k := SetupTestInput()
 	store := ctx.KVStore(k.StoreKey)
-	store.Delete(k.getSentDocumentsStoreKey(TestingDocument.Sender))
-	store.Delete(k.getReceivedDocumentsStoreKey(TestingDocument.Recipient))
+	store.Delete(k.getSentDocumentsStoreKey(TestingSender))
+	store.Delete(k.getReceivedDocumentsStoreKey(TestingRecipient))
```

Comment:
you shouldn't need this if you keep initializing store in every test with new `ctx` and `k`